eve/main.yml

```diff
@@ -7,5 +7,21 @@ branches:
 stages:
   pre-merge:
```

Comment:
As we talked about, I'll move the build into a separate step and call it in `TriggerStages` for the pre-merge step in a PR to add linting targets.